benchexec/runexecutor.py

```diff
@@ -692,8 +695,8 @@ def _setup_file_hierarchy_limit(
 
     # --- run execution ---
 
-    def execute_run(self, args, output_filename, stdin=None,
-                   hardtimelimit=None, softtimelimit=None, walltimelimit=None,
+    def execute_run(self, args, output_filename, error_filename=None, stdin=None, write_header=True,
```

Comment:
Inserting parameters in the middle breaks backwards compatibility.

Reply:
Ok, had not thought about this, should I change it for the internal call as well?

Reply:
For consistency, maybe yes.